apps/pp/serializers.py

```diff
@@ -235,9 +237,17 @@ class Meta:
 
 class AnnotationReportDeserializer(ResourceTypeSerializer):
     class Attributes(serializers.ModelSerializer):
+        def validate(self, data):
+            if data.get('reason') is SUGGESTED_CORRECTION and not data.get('comment'):
+                raise ValidationError({
+                    'comment': 'Comment is required for report "%s" reason' % SUGGESTED_CORRECTION}
+                )
+            return data
+
         class Meta:
             model = AnnotationReport
             fields = ('reason', 'comment')
+            extra_kwargs = {'comment': {'required': False, 'allow_blank': True}}
```

Comment:
I can't help but wonder - how is it possible that the model field `comment` is not set to `blank=true` and it's enough to set it in the serializer (apparently it just works!)?

Reply:
That's stange, I'd better set blank explicitly, before related edge-case bug comes :wink: